custom/icds_reports/tests/agg_tests/fixtures/pnc_forms.csv

```diff
@@ -66,3 +66,4 @@ form63,2018-04-01,0,2017-05-02,st2,2017-05-02,daf858cb-596b-443e-9b1f-0b52d6d4e3
 form63,2018-04-01,1,2017-05-02,st2,2017-05-02,daf858cb-596b-443e-9b1f-0b52d6d4e35f,child_13,1,,1,,,,,,,,1,1,,0,6,,s2,
 form64,2018-04-01,0,2017-05-03,st2,2017-05-02,daf858cb-596b-443e-9b1f-0b52d6d4e35f,child_14,1,,1,,,,,,,,1,1,,0,6,,s2,
 form64,2018-04-01,1,2017-05-03,st2,2017-05-02,daf858cb-596b-443e-9b1f-0b52d6d4e35f,child_15,1,,1,,,,,,,,1,1,,0,6,,s2,
+form65,2018-04-01,1,2017-05-03,st2,2017-05-02,f491263a-4846-4ffd-a64d-a6653c03dd03,child_16,1,,1,,,,,,,,1,1,,0,8,,s2,-100
```

Comment:
Form with negative value for `total_ifa_tablets_received` field

Reply:
It would be good to test normal values too, not just crazy ones. Especially good to also test the month logic (where it falls back to the previous month)